app/src/sharedTest/java/org/oppia/android/app/testing/NavigationDrawerActivityTest.kt

```diff
@@ -272,8 +277,185 @@ class NavigationDrawerActivityTest {
     }
   }
 
+  // Require opening the NavDrawer several times which is currently not supported on robolectric
```

Comment:
Can you file an issue for this and add a TODO here?

Reply:
Fix NavDrawer multiple times opening for Robolectric.

Reply:
#2535